contracts/osmo-liquid-pooler/src/error.rs

```diff
@@ -16,9 +16,6 @@ pub enum ContractError {
     #[error("{0}")]
     OverflowError(#[from] OverflowError),
 
-    #[error("Not clock")]
-    ClockVerificationError {},
-
```

Comment:
I think we need to add this:
```
#[error(transparent)]
ContractOperationError(#[from] ContractOperationError),
```

Reply:
Hm not really. The contract returns a NeutronError and this is done here

``impl From<ContractOperationError> for NeutronError {
    fn from(op_err: ContractOperationError) -> Self {
        NeutronError::Std(StdError::generic_err(op_err.to_string()))
    }
}``

In fact the error I removed (ClockVerificationError) was not being used anywhere of the same reason. If there's an error missing the contract would compile